suites/quincy/cephadm/test-haproxy-deployment.yaml

```diff
@@ -0,0 +1,266 @@
+#===============================================================================================
```

Comment:
Update name to `tier1-haproxy-deployment.yaml`

Reply:
done